### PR TITLE
Multi laser integration

### PR DESCRIPTION
**Pull request by jsongCMU**
Added AprilTag message publisher; when agent sees AprilTag, it'll publish AprilTagDetectionArray
Cleaned up / improved documentation on CMake, package.xml
Removed node publisher from sim launch file; will integrate version from SLAM_TOOLBOX soon

### CODE REVIEW

File: src/apriltag_publisher.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+// Agent info
+const std::vector<std::string> agent_names = {"agent1", "agent2"};
+const std::string base_frame_name = "base_link";
+// Tag info
+const std::string tag_name = "tag_";
+const size_t tag_amount = 20;
+// Topic info
+const std::string topic_name_prefix = "/robosar_agent_bringup_node/";
+const std::string topic_name_suffix = "/feedback/apriltag";
```
Comment (rachelzh9) [inline src/apriltag_publisher.cpp]: Can we make these rosparams? Bc I will also have to use them on slam_toolbox side to subscribe

Comment (jsongCMU) [reply src/apriltag_publisher.cpp]: Done

**Review by rachelzh9 — APPROVED** [review]
yey

**Review by rachelzh9 — APPROVED** [review]
YAY 2

**Pull request merged**